lib/hooks/queries/useTradeItemState.ts

```diff
@@ -67,13 +69,22 @@ export const useTradeItemState = (item: TradeItem) => {
       ?.data?.balance?.free.toString() ?? 0,
   );
 
+  const balances = {
+    poolBaseBalance: poolBaseBalance?.toString(),
+    poolAssetBalance: poolAssetBalance?.toString(),
+    traderBaseBalance: traderBaseBalance?.toString(),
+    traderAssetBalance: traderAssetBalance?.toString(),
+  };
+
   const query = useQuery(
     [
       id,
       tradeItemStateRootQueryKey,
+      poolAccountId,
```

Comment:
Could you add `queryClient.invalidateQueries([id, tradeItemStateRootQueryKey, account]);` to `useSubscribeBlockEvents`? That way this query will get invalidated when the pool account balance changes

Reply:
it get's invalidated when balances change without this, since `balances` are part of the query key (line 85). Is this really necessary. I tested this with app opened on two tabs, made a trade on one tab and trade values change on the second tab